apps/src/gamelab/GameLab.js

```diff
@@ -130,9 +162,59 @@ GameLab.prototype.reset = function (ignore) {
   this.tickIntervalId = 0;
   this.tickCount = 0;
 
+  /*
+  var divGameLab = document.getElementById('divGameLab');
+  while (divGameLab.firstChild) {
+    divGameLab.removeChild(divGameLab.firstChild);
+  }
+  */
```

Comment:
This is code held over from applab that clears out the div in case elements were created inside it. It has the added benefit of disconnecting event handlers that may have been attached. So far, we don't need this in gamelab. The canvas element from p5 is the only thing inserted under `divGameLab` and we ensure that it is removed through `p5.remove()`